cloud/linode/loadbalancers_test.go

```diff
@@ -285,6 +290,37 @@ func testCreateNodeBalancer(t *testing.T, client *linodego.Client, _ *fakeAPI) {
 	}
 
 	defer func() { _ = lb.EnsureLoadBalancerDeleted(context.TODO(), "linodelb", svc) }()
+	return nil
+}
+
+func testCreateNodeBalancerWithOutFirewall(t *testing.T, client *linodego.Client, f *fakeAPI) {
+	err := testCreateNodeBalancer(t, client, f, nil)
+	if !reflect.DeepEqual(err, nil) {
```

Comment:
No need to `DeepEqual` with nil, a simple comparison is sufficient. (same in the test below)